website/docs/docs/use-dbt-semantic-layer/dbt-semantic-layer.md

```diff
@@ -0,0 +1,133 @@
+
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+
+<VersionBlock firstVersion="1.3" >
+
+<span>&#8226;</span> Have a North American multi-tenant <a href="https://cloud.getdbt.com/">dbt Cloud</a> account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API. <br />
+<span>&#8226;</span> Have both your production and development environments running dbt version 1.3 or higher <br />
+<span>&#8226;</span> Use Snowflake data platform <br />
+<span>&#8226;</span> Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>">=1.3.0", "<1.4.0"</code> in your dbt project <br />
+<span>&#8226;</span> Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview">Metadata API</a> in the integrated tool to import metric definitions <br />
+<span>&#8226;</span> Recommended - Review the <a href="https://docs.getdbt.com/docs/building-a-dbt-project/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/getting-started-with-the-dbt-semantic-layer">Getting started with the dbt Semantic Layer</a> blog <br />
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.2">
+
+<span>&#8226;</span> Have a North American multi-tenant <a href="https://cloud.getdbt.com/">dbt Cloud</a> account. Developer accounts will be able to query the Proxy Server using SQL, but will not be able to browse pre-populated dbt metrics in external tools, which requires access to the Metadata API. <br /> 
+<span>&#8226;</span> Have both your production and development environments running dbt version 1.2 (latest) <br />
+<span>&#8226;</span> Use Snowflake data platform <br />
+<span>&#8226;</span> Install the <a href="https://hub.getdbt.com/dbt-labs/metrics/latest/">dbt metrics package</a> version <code>">=1.3.0", "<1.4.0"</code> in your dbt project <br />
+<span>&#8226;</span> Set up the <a href="https://docs.getdbt.com/docs/dbt-cloud/dbt-cloud-api/metadata/metadata-overview">Metadata API</a> in the integrated tool to import metric definitions <br />
+<span>&#8226;</span> Recommended - Review the <a href="https://docs.getdbt.com/docs/building-a-dbt-project/metrics">dbt metrics page</a> and <a href="https://docs.getdbt.com/blog/getting-started-with-the-dbt-semantic-layer">Getting started with the dbt Semantic Layer</a> blog <br />
+
+</VersionBlock>
+    
+<---!
+<Snippet src="sl-prerequisites" />
+--->
```

Comment:
This block is also causing the build to fail. The `<---!` `--->` has to be removed. If trying to comment out this snippet, it should be adjusted to:
```suggestion
<!--
<Snippet src="sl-prerequisites" />
-->
```